forte/data/data_pack.py

```diff
@@ -453,19 +460,88 @@ def groups(self):
     def groups(self, val):
         self._groups = val
 
-    def get_span_text(self, begin: int, end: int) -> str:
+    def get_payload_at(
+        self, modality: str, payload_index: int
```

Comment:
Maybe we can reuse the `Modality` enum defined in `top.py` instead of `string` for `modality`?

Reply:
We probably would also move `Enum` out from the `top.py` file? On one hand, we can keep `top.py` mainly for ontology only, on the other, it would make importing these library-level Enums easier.

Reply:
created another file `forte/data/modality.py` to keep `Modality`